Mathlib/NumberTheory/NumberField/FinitePlaces.lean

```diff
@@ -150,20 +151,22 @@ theorem vadicAbv_intCast_le_one (n : ℤ) : vadicAbv v n ≤ 1 := by
 open FinitePlace
 
 /-- The `v`-adic norm of an integer is at most 1. -/
-theorem norm_le_one (x : 𝓞 K) : ‖embedding v x‖ ≤ 1 := by
+theorem norm_le_one (x : 𝓞 (WithVal (v.valuation K))) : ‖embedding v x‖ ≤ 1 := by
```

Comment:
I think all this should go inside the `FinitePlace` namespace because `norm_le_one` is false if `v` is archimedean. If you think this should go to another PR I understand, but I would then open that one before resuming this. Insofar I am concerned, keeping it here is OK.